libs/ui-toolkit/src/components/button/button.tsx

```diff
@@ -28,11 +29,12 @@ const getClasses = (
   variant: CommonProps['variant'],
   paddingLeftProvided: boolean,
   paddingRightProvided: boolean,
-  borderWidthProvided: boolean
+  borderWidthProvided: boolean,
```

Comment:
suggestion (non-blocking): I don't think we should add these bool checks for styles. I'd prefer to just update the design in the token app to use whatever base button styles we have for now. Theres plenty of other changes to address so we can consider this (and whether or not we use the tailwind override lib) in a further iteration.

(same goes for the other changes in this file)